java-checks-test-sources/src/main/java/checks/PseudoRandomCheck.java

```diff
@@ -42,12 +46,24 @@ void fun() {
     RandomStringUtils randomStringUtils = new RandomStringUtils();
     String rand8 = randomStringUtils.random(1); // Noncompliant
     String rand9 = RandomStringUtils.random(1); // Noncompliant
+    rand9 = random(1); // Noncompliant
```

Comment:
This is a FN - previously we didn't report on statically imported methods because we used the receiver expression to determine the class of the method